from elasticsearch import NotFoundErrorfrom application import celery, es, loggerfrom application.utils.mappings import publication_mappingsfrom application.tasks.authors_scraper import t_authors_scraperfrom application.utils.decorators import celery_exception_handler@celery.task(bind=True, name="starter", max_retries=3)@celery_exception_handler(ConnectionError, NotFoundError)def t_starter(self):    resd = {"status": "ok"}    # ignore 400 cause by IndexAlreadyExistsException when creating an index    resp = es.indices.create(        index="publication",        body={"mappings": publication_mappings},        ignore=400    )    logger.info(f'Create indices response: {resp}')    t_authors_scraper.apply_async()    return resd